File: in_Python/2070A_fizzbuzz_remixed.py
t = int(input())
for i in range(t):
    num = int(input())
    
    count = 0
    
    if (num <= 2):
        count = count + num + 1
    elif (num > 2 ):
        count = count + 3
    diff = int(num/15)
    if (diff > 0):
        count = count + (diff-1)*3
        if (num >= diff*15):
            count+=1
        if (num >= (diff*15)+1):
            count+=1
        if (num >= (diff*15)+1):
            count+=1
    print(count)

File: in_C++/2070A_fizzbuzz_remixed.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;
    while(t--) {
        int num;
        cin >> num;
        int count = 0;
        if (num <=2) {
            count = count + num +1;
        }
        else if (num > 2) {
            count = count + 3;
        }
        int diff = num / 15;
        
        if (diff > 0) {
            count = count + (diff-1)*3;
            if (num >= diff*15) {
                count++;
            }
            if (num >= (diff*15 )+ 1) {
                count++;
            }
            if (num >= (diff*15 )+ 2) {
                count++;
            }
        }
        cout << count << endl;
    }
    return 0;
}